src/test/java/org/broadinstitute/hellbender/utils/baq/BAQUnitTest.java

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.hellbender.utils.baq;
+
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public final class BAQUnitTest extends BaseTest{
+
+    @Test //regression test for https://github.com/broadinstitute/gatk/issues/1234
+    public void testGetReferenceWindowForRead() throws Exception {
```

Comment:
Test case needs a more specific name, since it's not testing `BAQ.getReferenceWindowForRead()` comprehensively, it's just testing truncation at contig start. How about `testGetReferenceWindowForReadStopsAtContigStart`